nymea-app/ui/mainviews/WallboxView.qml

```diff
@@ -46,70 +46,94 @@ MainViewBase {
         shownInterfaces: ["evcharger"]
     }    
 
-    SwipeView {
-        id: swipeView
+    Rectangle {
         anchors.fill: parent
-        anchors.horizontalCenter: parent.horizontalCenter
-        currentIndex: pageIndicator.currentIndex
-        visible: wallboxDevices.count != 0
+        color: app.foregroundColor
 
-        Repeater {
+        SwipeView {
+            id: swipeView
             anchors.fill: parent
             anchors.horizontalCenter: parent.horizontalCenter
-            model: wallboxDevices
-
-            delegate: Item {
-                id: wallboxDelegate
-                height: swipeView.height
-                width: swipeView.width
-
-                property Thing thing: wallboxDevices.get(index)
-                property State powerState: thing.stateByName("power")
-                property State maxChargingCurrentState: thing.stateByName("maxChargingCurrent")
-
-                Rectangle {
-                    width: swipeView.width
+            currentIndex: pageIndicator.currentIndex
+            visible: wallboxDevices.count != 0
+
+            Repeater {
+                anchors.fill: parent
+                anchors.horizontalCenter: parent.horizontalCenter
```

Comment:
fwiw, a Repeater is a non-visual item and doesn't require any size/position information